xmtp_mls/src/storage/encrypted_store/group_message.rs

```diff
@@ -24,12 +27,14 @@ pub struct StoredGroupMessage {
     pub decrypted_message_bytes: Vec<u8>,
     /// Time in nanoseconds the message was sent.
     pub sent_at_ns: i64,
-    /// Group Message Kind Enum
+    /// Group Message Kind Enum: 1 = Application, 2 = MembershipChange
     pub kind: GroupMessageKind,
     /// The ID of the App Installation this message was sent from.
     pub sender_installation_id: Vec<u8>,
     /// Network wallet address of the Sender
     pub sender_account_address: String,
+    /// Enum: 1 = Published, 2 = Unpublished
```

Comment:
Another nit, it might be better to not document the number value in all of these comments, so there is a single source of truth in the enum itself, in the event more variants are added or existing variants are updated.

Reply:
removed.  Folks will have to jump to the `DeliveryStatus` itself to find the truth.